LICENSE

```diff
@@ -281,4 +281,109 @@ YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE WITH ANY OTHER
 PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE
 POSSIBILITY OF SUCH DAMAGES.
 
-                     END OF TERMS AND CONDITIONS
\ No newline at end of file
+                     END OF TERMS AND CONDITIONS
+
+
+Business Source License 1.1
+
+License text copyright (c) 2017 MariaDB Corporation Ab, All Rights Reserved.
+"Business Source License" is a trademark of MariaDB Corporation Ab.
+
+-----------------------------------------------------------------------------
+
+Parameters
+
+Licensor:             Morpho Association
+
+Licensed Work:        Morpho Blue Core
+                      The Licensed Work is (c) 2023 Morpho Association
+
+Additional Use Grant: Any uses listed and defined at
+                      morpho-blue-core-license-grants.morpho.eth
```

Comment:
this i guess must be updated

Reply:
Ah yes. Not sure how to do it, especially the "change date" which should be void I think. I guess it should be explained that the users can choose the license as well